content/partials/specs/versions-macos-m1-xcode-14-2.md

```diff
@@ -73,6 +73,7 @@ UNITY_HOME: /Applications/Unity/Hub/Editor/2021.3.7f1/Unity.app
 
 ## Xcode versions
 
+- 14.3 (14E5197f) `/Applications/Xcode-14.3.app`
```

Comment:
`edge` should also point to `14.3`

Reply:
@helinanever unfortunately this is not possible before further developments, see my comment in the relevant Trello task. We should properly fail builds with `mac_pro` and `edge` or implement logic to have separate `edge` resolvers per platform.

Reply:
Makes sense.